bin.js

```diff
@@ -1,8 +1,9 @@
-/* eslint-disable no-console */
+#!/usr/bin/env node
+
 const integration = require('./index')
 integration.execute()
   .then((msg) => {
-    console.log(JSON.parse(msg, null, 2))
+    console.log(msg)
```

Comment:
console.log does not always show all nested objects. any reason you've removed the stringify call?

Reply:
yes, i got a JSON parse error